regml.py

```diff
@@ -964,15 +1042,20 @@ def apply_through(cfr_title, cfr_part, start=None, through=None,
                                         pretty_print=True,
                                         xml_declaration=True,
                                         encoding='UTF-8')
-        new_path = os.path.join(
-            os.path.dirname(regulation_file),
-            os.path.basename(notice_file))
+        if doc_counts[notice.document_number] == 1:
+            new_path = os.path.join(
+                os.path.dirname(regulation_file),
+                os.path.basename(notice_file))
+        elif doc_counts[notice.document_number] > 1:
```

Comment:
Is this `elif` clause needed here, or can this just be an `else`? If there are no `doc_counts` what should happen here?